python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py

```diff
@@ -688,6 +688,60 @@ def intersect(
         self.set_visible(shape2, False)
         return self.add_object(OBJECT_FACTORY.create_object(data, self))
 
+    def extrude(
+        self,
+        name: str = "",
+        shape: str | int = None,
+        direction: List[float] = [0, 0, 1],
+        length_fwd: float = 10,
+        length_rev: float = 0,
+        solid: bool = False,
+        color: Optional[str] = None,
```

Comment:
Although I see the correct color when you extrude in the video clip on this PR but still it'd be nicer to use `"#808080"` as fallback here